### ISSUE TITLE
plotly_streaming_ethernet.h: No such file or directory

### ISSUE THREAD

**Issue by andershf** [plotly/arduino-api#36]
Following this tutorial: https://plot.ly/arduino/dht22-temperature-tutorial/

Extracted the library (arduino-api-master) under Arduino/libraries/ and renamed it to "Plotly"

Get the following error:
dht22_ethernet.ino:3:39: fatal error: plotly_streaming_ethernet.h: No such file or directory
compilation terminated.
Error compiling.

**Comment by alexander-daniel** [comment]
Please make sure you have installed the libraries to the correct locations. Give https://github.com/plotly/arduino-api/issues/3 this solution a try and let me know!
Thanks!

**Comment by andershf** [comment]
Thanks for the quick response @alexander-daniel 

Here's a screenshot of the location: http://goo.gl/esM6dW

**Comment by alexander-daniel** [comment]
Great, thanks. Try moving the *individual* plotly libraries outside of the `Plotly` folder. Let me know if that works. Each folder within that `Plotly` folder is its own library, so it needs to go up one level!

Let me know if that works.

**Comment by andershf** [comment]
Thanks @alexander-daniel now I get the error as in issue #3 

Since I'm using Windows I didn't quite understand the solution in the thread. Could you please give me a quick guide?

**Comment by prasanthratso** [comment]
@alexander-daniel i am facing a similar problem as what @andershf  is facing now,im using a windows  pc ,please give me a possible solution .thanks in advance

**Comment by alexander-daniel** [comment]
Yup! I'll give it a go on our windows machine today and let you know. Thanks!

**Comment by prasanthratso** [comment]
Hi
i am trying to use plotly for streaming my sensor data through GSM/GPRS
shield ,i am currently using a ARDUINO IDE 1.6.0 in my windows 7, 32 bit OS
.As i tried to compile my program using the same it returned an error
stating that avr/dtostrf.h: No such file or directory.i took some
suggestion to fix this problem by installing IDE 1.0.6 and the program for
streaming through Ethernet worked absolutely fine, but i still keep facing
a problem in compiling the example program for streaming through  GPRS that
you have provided in Github .kindly help me to fix this problem ,i am
eagerly waiting for solution for this problem from your team, thanks in
advance

**Comment by alexander-daniel** [comment]
@prasanthratso So ethernet compiles and works,but GPRS does not?

What is the error output you're receiving when you try to compile the GPRS sketch on 1.0.6?

**Comment by prasanthratso** [comment]
@alexander-daniel it returns me the below errors


C:\Program Files\Arduino\libraries\GSM\GSM3ShieldV1ModemVerification.cpp: In member function 'String GSM3ShieldV1ModemVerification::getIMEI()':
C:\Program Files\Arduino\libraries\GSM\GSM3ShieldV1ModemVerification.cpp:72: error: conversion from 'int' to 'String' is ambiguous
C:\Program Files\Arduino\hardware\arduino\cores\arduino/WString.h:61: note: candidates are: String::String(const __FlashStringHelper*)
C:\Program Files\Arduino\hardware\arduino\cores\arduino/WString.h:59: note:                 String::String(const char*)

**Comment by michaelproino** [comment]
Hi, I have the same problem ;
No such file or directory compilation terminated.
Error compiling.

I add the library in zip format, correctly but when I try to run example code, this error occurs, can you help me?
